CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Custom views/GiniBarButton.swift

```diff
@@ -192,6 +192,18 @@ public final class GiniBarButton {
 
         return attributes
     }
+
+    public func setContentHuggingPriority(_ priority: UILayoutPriority, for axis: NSLayoutConstraint.Axis) {
+        stackView.setContentHuggingPriority(priority, for: axis)
+        titleLabel.setContentHuggingPriority(priority, for: axis)
+        imageView.setContentHuggingPriority(priority, for: axis)
+    }
+
+    public func setContentCompressionResistancePriority(_ priority: UILayoutPriority, for axis: NSLayoutConstraint.Axis) {
+        stackView.setContentCompressionResistancePriority(priority, for: axis)
+        titleLabel.setContentCompressionResistancePriority(priority, for: axis)
+        imageView.setContentCompressionResistancePriority(priority, for: axis)
+    }
```

Comment:
[clarification] We need to make sure we did not break anything related to GiniBarButton by adding this.

Reply:
All should be good, I have done it like methods to update only where it's needed. So we will use it only on Skonto for dynamic layout